lib/esm/generated/telemetry.d.ts

```diff
@@ -127,6 +127,14 @@ export declare type TelemetryConfigurationEvent = CommonTelemetryProperties & {
              * Whether sessions across subdomains for the same site are tracked
              */
             track_session_across_subdomains?: boolean;
+            /**
+             * Whether resources are tracked
+             */
+            track_resources?: boolean;
+            /**
+             * Whether long tasks are tracked
+             */
+            track_long_task?: boolean;
```

Comment:
1 - We do not use thresholds nor specify sampling rate/thresholds per metric on the browser SDK. We currently only have sampling rate that applies to all metrics and then booleans to disable specific metrics from being collected. It is an interesting idea to explore however, so I have reached out to the team to discuss it

2 - Great point! I think it makes sense as a different PR as this would effect mainly mobile SDKs and the aim here was just to add the missing flags of browser SDK.